tests/phpunit/Process/Runner/TestRunConstraintCheckerTest.php

```diff
@@ -202,4 +202,85 @@ public function test_run_passes_on_exactly_covered_min_msi(): void
 
         $this->assertTrue($constraintChecker->hasTestRunPassedConstraints());
     }
+
+    /** @dataProvider is_msi_over_minimum_msi_provider */
```

Comment:
We generally use multi line comments for dataproviders, even if they are the only comment.

Reply:
Done